src/commands/cont.rs

```diff
@@ -9,8 +9,46 @@ pub struct ContinueCommand;
 
 impl ContinueCommand {
     pub async fn execute(api_client: impl ApiClient) -> ResultWithDefaultError<()> {
+        let running_entry_stop_time = Utc::now();
+        let running_time_entry = api_client.get_running_time_entry().await;
+        match &running_time_entry {
+            Err(error) => {
+                println!(
+                    "{} {:?}",
+                    "Couldn't retrieve running time entry.".red(),
+                    error
+                );
+                return Ok(());
+            }
+            Ok(value) => {
+                if let Some(time_entry) = value {
+                    if let Err(stop_error) = api_client
+                        .update_time_entry(
+                            time_entry.as_stopped_time_entry(running_entry_stop_time),
+                        )
+                        .await
+                    {
+                        println!(
+                            "{} {:?}",
+                            "Couldn't stop running time entry.".red(),
+                            stop_error
+                        );
+                        return Ok(());
+                    }
+                    println!("Running time entry stopped")
+                }
+            }
+        }
+
         let time_entries = api_client.get_time_entries().await?;
-        match time_entries.first() {
+        // Don't continue a running entry that was just stopped.
+        let continue_entry_index = if let Ok(None) = running_time_entry {
+            0
+        } else {
+            1
+        };
```

Comment:
```suggestion
        let continue_entry_index = match running_time_entry {
            Ok(None) => 0,
            _ => 1
        };
```

`match` can simplify these statements slightly